packages/settings-library/src/settings_library/tracing.py

```diff
@@ -0,0 +1,18 @@
+from pydantic import AnyUrl, Field
+
+from .base import BaseCustomSettings
+
+
+class TracingSettings(BaseCustomSettings):
+    TRACING_ZIPKIN_ENDPOINT: AnyUrl = Field(
+        "http://jaeger:9411", description="Zipkin compatible endpoint"
+    )
+    TRACING_THRIFT_COMPACT_ENDPOINT: AnyUrl = Field(
+        "http://jaeger:5775",
+        description="accept zipkin.thrift over compact thrift protocol (deprecated, used by legacy clients only)",
+    )
+    TRACING_CLIENT_NAME: str = Field(
+        "undefined-tracing-client-name",
+        description="Name of the application connecting the tracing service",
```

Comment:
So this will be for example `director-v2` or `webserver`, right?

Reply:
yes. I corrected the title of the PR.